_posts/2019-04-26-compatibility-table.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Ruby & Rails Compatibility Table"
+date: 2019-04-26 13:00:00
+categories: ["ruby", "rails", "versions"]
+author: "etagwerker"
+---
+
+This is a short post to show the compatibility between [Ruby on Rails](https://rubyonrails.org)
+and [Ruby](https://www.ruby-lang.org/en/) across different versions. In the
+process of upgrading really old applications to more modern versions of Ruby and
+Rails we have ran into a lot of these combinations.
+
+<!--more-->
+
+<table id="ruby-rails-compatibility">
+  <thead>
+    <tr>
+      <td>Rails<br/>Version</td>
+      <td>Required<br/>Ruby<br/>Version</td>
```

Comment:
@ercohen14 You can link Ruby version to https://www.ruby-lang.org/en/downloads/releases/